src/snipe/api/reference_QC.py

```
@@ -5,8 +5,17 @@
 
 import numpy as np
 from scipy.optimize import OptimizeWarning, curve_fit
-from snipe.api import SnipeSig
+from snipe.api.snipe_sig import SnipeSig
 from snipe.api.enums import SigType
+import os
+import requests
+from tqdm import tqdm
+import cgi
```

Ruff

`cgi` imported but unused

Remove unused import: `cgi`

(F401)